backend/lib/routes/index.js

```diff
@@ -21,7 +21,8 @@ module.exports = {
   '/namespaces/:namespace/shoots': require('./shoots'),
   '/namespaces/:namespace/tickets': require('./tickets'),
   '/namespaces/:namespace/cloud-provider-secrets': require('./cloudProviderSecrets'),
-  '/namespaces/:namespace/members': require('./members')
+  '/namespaces/:namespace/members': require('./members'),
+  '/namespaces/:namespace/resource-quotas': require('./resourceQuotas')
```

Comment:
In the k8s api the there is no `-`. I know we have them also in `cloud-provider-secrets` but not in `cloudprofiles`.